sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java

```diff
@@ -181,6 +192,30 @@ public CreateNewCaseSteps(WebDriverHelpers webDriverHelpers, CaseService caseSer
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(EditCasePage.REPORT_DATE_INPUT);
           webDriverHelpers.clickOnWebElementBySelector(CASE_SAVED_POPUP);
         });
+
+    When(
+        "^I create a new case for contact with specific data for DE$",
+        () -> {
+          caze = caseService.buildGeneratedCaseDE();
+          fillDateOfReport(caze.getDateOfReport(), Locale.GERMAN);
+          selectCaseOrigin(caze.getCaseOrigin());
+          fillExternalId(caze.getExternalId());
+          selectResponsibleRegion(caze.getResponsibleRegion());
+          selectResponsibleDistrict(caze.getResponsibleDistrict());
+          selectResponsibleCommunity(caze.getResponsibleCommunity());
+
+          selectPlaceOfStay(caze.getPlaceOfStay());
+          fillPlaceDescription(caze.getPlaceDescription());
+          selectPresentConditionOfPerson(caze.getPresentConditionOfPerson());
+          fillDateOfSymptomOnset(caze.getDateOfSymptomOnset(), Locale.GERMAN);
+          webDriverHelpers.selectFromCombobox(
+              CASE_DISEASE_VARIANT_COMBOBOX, "Sequenzierung ausstehend");
```

Comment:
If there are multiple variants that can be chosen from the combobox, it might me good to vary the data that test inputs.